### ISSUE TITLE
Add documentation for upgrade of database

### ISSUE THREAD

**Issue by tpokorra** [mosparo/mosparo#29]
Thanks for this nice project!

While upgrading to 0.3.9, I realized I needed to apply the doctrine database updates.

It might be worth adding an upgrade section to the documentation (https://github.com/mosparo/documentation/tree/master/docs), including this hint:

```
./bin/console doctrine:migrations:migrate
```

**Comment by zepich** [comment]
Hi @tpokorra

Thank you very much for your feedback!

Did you have some issues after updating? In general, the database updates should be executed on the first request to mosparo, for example, when you open mosparo in your browser.

But it's good feedback from you. The information about updating mosparo is hidden in the [administration section](https://documentation.mosparo.io/docs/administration/updates) of the documentation. As a user, I would not look for it there, to be honest. Additionally, it can help show the command to migrate the database manually.

I will update the documentation.

Additionally to installing the database updates, you have to clear the cache as well:

```
./bin/console cache:clear
```

**Comment by tpokorra** [comment]
Hello @zepich 
Thank you for your reply!
Ah, now I see the update documentation!

I got a 500 error, which meant the update could not run automatically.
I was on 0.3.6 before, and upgraded straight to 0.3.9. Perhaps that was the reason?

```
{"message":"Uncaught PHP Exception PDOException: \"SQLSTATE[42S02]: Base table or view not found: 1146 Table 'tim00_captcha.sessions' doesn't exist\" at /home/.../mosparo/vendor/symfony/http-foundation/Session/Storage/Handler/PdoSessionHandler.php line 667","context":{"exception":{"class":"PDOException","message":"SQLSTATE[42S02]: Base table or view not found: 1146 Table 'tim00_captcha.sessions' doesn't exist","code":42,"file":"/home/.../mosparo/vendor/symfony/http-foundation/Session/Storage/Handler/PdoSessionHandler.php:667"}},"level":500,"level_name":"CRITICAL","channel":"request","datetime":"2023-04-03T06:17:16.193120+02:00","extra":{}}
```

Thanks for the hint about clearing the cache, I was not aware of that.

**Comment by zepich** [comment]
Hi @tpokorra

Thank you very much for your report. I've tried it myself and had the same issue. The web update process needed the session to finalize the upgrade, while the table to read the session from did not exist.

I've added a fix and released v0.3.10. Updating from v0.3.8 or before to v0.3.10 will not have this problem anymore.

I will update the documentation to show the different options better for updating mosparo.
